src/django_components/component.py

```diff
@@ -318,6 +349,7 @@ def render(
                 # See https://github.com/EmilStenstrom/django-components/issues/280#issuecomment-2081180940
                 "component_vars": {
                     "is_filled": slot_bools,
+                    "attrs": attrs,
```

Comment:
I exposed the parsed attributes via `{{ component_vars.attrs }}`. Also, I made `attrs` immutable.